wp-content/themes/engage/assets/scss/components/_teammates.scss

```diff
@@ -45,7 +45,8 @@
 	}
 }
 
-.mate__name {
+.mate__name--full, a.mate__name--full:hover,
```

Comment:
You'll probably be able to remove this `a.mate__name--full:hover` after making the html change.